x-pack/plugin/analytics/src/main/java/org/elasticsearch/xpack/analytics/multiterms/MultiTermsAggregationBuilder.java

```diff
@@ -154,7 +155,7 @@ public boolean supportsSampling() {
     }
 
     @Override
-    public boolean supportsParallelCollection() {
+    public boolean supportsParallelCollection(ToLongFunction<String> fieldCardinalityResolver) {
```

Comment:
In a follow up we can enable parallel collection support here too. Similar to how enable it for terms agg.

Reply:
yep, that was on my radar too.